src/vision/status_quo/barbara_wants_to_use_ghostcell.md

```diff
@@ -0,0 +1,274 @@
+# Barbara wants to use GhostCell-like cell borrowing with futures
+
+## 🚧 Warning: Draft status 🚧
+
+This is a draft "status quo" story submitted as part of the
+brainstorming period. It is derived from real-life experiences of
+actual Rust users and is meant to reflect some of the challenges that
+Async Rust programmers face today.
+
+If you would like to expand on this story, or adjust the answers to
+the FAQ, feel free to open a PR making edits (but keep in mind that,
+as they reflect peoples' experiences, status quo stories [cannot be
+wrong], only inaccurate). Alternatively, you may wish to [add your own
+status quo story][htvsq]!
+
+## The story
+
+Barbara quite likes using statically-checked cell borrowing.  This is
```

Comment:
It took me a second to figure out that the cell is the first sentence refers to the cells like `std::cell::{Cell, RefCell}`. 

Perhaps you could rephrase this to "Barbara would like to use a form of mutable memory location like `std::cell::Cell` that are statically checked and guaranteed to not panic at runtime".

Reply:
I think "cell" is the correct terminology in the Rust world. If I Google "rust cell" I get Cell, RefCell, UnsafeCell etc. However I will add a clarification in a separate sentence.